docs/resources/action.md

```diff
@@ -81,7 +81,7 @@ resource "auth0_action" "my_action" {
 Required:
 
 - `id` (String) The trigger ID.
-- `version` (String) The trigger version.
```

Comment:
This does not control the runtime, but it does regulate which runtimes are supported. 

>Make sure the `version` supports the desired node `runtime`

Is this actually actionable with our current documentation?